dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -146,9 +150,27 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
                   ))}
                 </select>
               </div>
-              {this.props.enableBasicForm ? this.renderBasicForm() : this.renderAdvancedForm()}
+              <div className="margin-t-big" hidden={!this.props.enableBasicForm}>
+                <div className="Tabs">
+                  <div
+                    className={`Tabs__Tab ${this.state.showBasicForm ? "Tabs__Tab-active" : ""}`}
+                  >
+                    <button type="button" onClick={this.setBasicForm(true)}>
+                      Basic
+                    </button>
+                  </div>
+                  <div
+                    className={`Tabs__Tab ${this.state.showBasicForm ? "" : "Tabs__Tab-active"}`}
+                  >
+                    <button type="button" onClick={this.setBasicForm(false)}>
+                      Advanced
+                    </button>
+                  </div>
+                </div>
+              </div>
+              {this.state.showBasicForm ? this.renderBasicForm() : this.renderAdvancedForm()}
```

Comment:
But this is inside the div which is hidden isn't it? (ie. it appears that (1) the normal advanced form is only rendered within the new tabs which should only appear if the feature is on, and (2) the surrounding div is hidden if the feature is not enabled - but I could have missed something - anyway, +1 for better readability)

Reply:
ah, I think that you think that the `Tabs` include the form within but those are separate elements. In any case, now it should be more readable.

Reply:
I agree with @absoludity. This is a potential accessibility issues because the content of the tabs is fully disconnected from the tabs navigation. These elements should be scoped under a section element and connect the tabs triggers with the content.

You have an example of accessible tabs on https://dev.to/link2twenty/accessibility-first-tabs-ken. For now, I believe you can ignore advanced features like arrow keys navigation. Using the tab key and the enter/space should be enough for the navigation.